roles/confluent.kafka_connect/tasks/main.yml

```diff
@@ -103,3 +103,9 @@
     name: "{{kafka_connect.service_name}}"
     enabled: "{{kafka_connect.systemd.enabled}}"
     state: "{{kafka_connect.systemd.state}}"
+
+- name: Register connector configs and remove deleted connectors
+  confluent_connectors:
+    connect_url: "{{kafka_connect_http_protocol}}://{{kafka_connect.rest_server}}:{{kafka_connect_rest_port}}/connectors"
```

Comment:
I think I asked in the other PR if we could just use `kafka_connect[0]` here?
Also have you tested if kafka_connect.connectors does not exist?

And just me being picky, the kafka_connect dictionary is getting loaded w a bunch of stuff, can can you change kafka_connect.connectors to be its own dictionary - kafka_connect_connectors

Reply:
OK, no problem. I'll make those changes ASAP.